VideoApp/VideoApp/Video/Room/ConnectOptionsFactory.swift

```diff
@@ -26,6 +26,15 @@ import TwilioVideo
         videoTracks: [TwilioVideo.LocalVideoTrack]
     ) -> ConnectOptions {
         ConnectOptions(token: accessToken) { builder in
+            var videoBitrate: UInt {
+                switch self.appSettingsStore.videoCodec {
+                case .h264: return 1_200
+                case .vp8: return 1_200
+                case .vp8SimulcastVGA: return 0
+                case .vp8SimulcastHD: return 1_600
+                }
+            }
```

Comment:
I'm thinking I'll probably replace the extensions with something like this in the future. This is actually a pretty interesting factory because it does so much.